stdlib/regex/src/main/ballerina/Ballerina.toml

```diff
@@ -0,0 +1,3 @@
+[project]
+org-name = "ballerina"
+version = "0.0.0"
```

Comment:
Shall we add a proper version number here ?

Reply:
Added